.changeset/witty-laws-join.md

```diff
@@ -0,0 +1,5 @@
+---
+"replayio": patch
+---
+
+Fixed a potential data corruption issue related to the `recrdings.log` updates
```

Comment:
@hbenl feel free to tweak this if you think this is inaccurate
```suggestion
Fixed a potential data corruption issue related to the `recordings.log` updates
```

oops, i made a type 🤐